benchmark/Project.toml

```diff
@@ -11,7 +11,10 @@ FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
```

Comment:
Is Oceananigans supposed to be here? If not how come things were working before? Or was it broken?

Reply:
I don't think we want Oceananigans here... we want to use 

```julia
push!(LOAD_PATH, joinpath(@__DIR__, ".."))
```

right at the top of the script (just like the docs):

https://github.com/CliMA/Oceananigans.jl/blob/29bdcb32cda5071ad981ddd4e97d22d02ced8001/docs/make.jl#L1

This is a convenient pattern for getting the benchmarks to run with the "local" Oceananigans code, while having a separate `Project.toml` for the benchmarks.

Reply:
@glwagner does anything in Manifest.toml need to be changed as well?